### DISCUSSION TITLE
Any chance of multi-gpu / multi-node support in future?

### DISCUSSION THREAD

**Discussion by jlinBE** [NVIDIA/warp#32] [General]
Is this on the roadmap?  It's a shame for Warp to be limited to a single graphics card.

**Comment by mmacklin** [comment]
Hi @jlinBE, we added MGPU support in `0.4x` and have just updated documentation on how to use MGPU: https://nvidia.github.io/warp/_build/html/modules/runtime.html#devices

Thanks,
Miles
